distribution/ddf-common/src/main/resources/etc/log4j2.config.xml

```diff
@@ -31,12 +29,12 @@
         <Syslog name="syslog" facility="AUTH" host="localhost" protocol="UDP" port="514"/>
 
         <Console name="stdout" target="SYSTEM_OUT">
-            <PatternLayout pattern="%d{ABSOLUTE} | ${log-pattern}"/>
+            <PatternLayout pattern="%d{ISO8601} | ${log-pattern}"/>
         </Console>
 
         <RollingFile name="out" append="true"
-                     fileName="${sys:karaf.data}/log/ddf.log"
-                     filePattern="${sys:karaf.data}/log/ddf.log-%d{yyyy-MM-dd-HH}-%i.log.gz">
+                     fileName="${sys:karaf.data}/log/${sys:org.codice.ddf.system.branding}.log"
+                     filePattern="${sys:karaf.data}/log/${sys:org.codice.ddf.system.branding}.log-%d{yyyy-MM-dd-HH}-%i.log.gz">
```

Comment:
Using a property here makes a https://github.com/codice/alliance/commit/9adc0b879e15db4a0f7d292497f86e19066a71bd unecessary. These changes should be removed in downstream projects.